calico/felix/config.py

```diff
@@ -248,6 +248,12 @@ def __init__(self, config_path):
                            "spoofing their source IP.  (For example, "
                            "unprivileged containers.)",
                            False, value_is_bool=True)
+        self.add_parameter("ClusterGUID", "Unique cluster identifier",
+                           "baddecaf")
+        self.add_parameter("UsageReportingEnabled",
+                           "If set to true, the cluster existence will"
```

Comment:
Fixed.

Changed to self.add_parameter("UsageReportingEnabled",
                           "If set to true, periodically report cluster "
                           "version, hostname, size and guid to "
                           "projectcalico.org.  Receive version deprecation / "
                           "security warnings.","